src/admin-orderable-print/_orderable-print.scss

```diff
@@ -0,0 +1,4 @@
+.data-download {
+    font-weight: normal;
+    font-size: 14px;
+}
```

Comment:
I am not sure, but I think that one scss file is enough as the scope for scss files is whole application. Could you please check?

Reply:
I think that we can keep like this and in the future try to move this into global scss file.